### DISCUSSION TITLE
Huion H610 non-X/Pro

### DISCUSSION THREAD

**Discussion by RudiusExCursori** [OpenTabletDriver/OpenTabletDriver#3338] [General]
I am not sure the difference  between to original, X, and Pro. but the original is not supported but the X/Pro is, is there a chance to have it added as i do not currently know how to do it myself? or can someone link to where i could learn to do it myself?

**Comment by jamesbt365** [comment]
There is currently no documentation for creating configurations, however what I can say is `help > export diagnostics` and `tablet > device string reader` are useful, but if you can't figure anything out then our [Discord Server](https://discord.com/invite/9bcMaPkVAR) is an option.

**Comment by RudiusExCursori** [comment]
I have played around with the default drivers and the the first H610 Pro Drivers are functionally the same and the regular H610 or maybe even just the same driver. so the drivers for it are already in OpenTabletDrivers it just isn't recognizing that that's what the H610 needs.

[Detect:Info]	No tablets were detected.
[Detect:Info]	Searching for tablets...
[Detect:Info]	No tablets were detected.
[Detect:Info]	Searching for tablets...
[Detect:Info]	No tablets were detected.
     TABLET UNPLUGGED
[WinUSB:Info]	Cannot create device for '\\?\usb#vid_0408&pid_5425&mi_02#7&2c366651&0&0002#{dee824ef-729b-4a0e-9c14-b7117d33a817}'
[RootHub:Info]	Debounced HidSharpDeviceRootHub's DevicesChanged event
     TABLET PLUGGED BACK IT
[WinUSB:Info]	Cannot create device for '\\?\usb#vid_256c&pid_006e&mi_00#7&4caab7b&0&0000#{dee824ef-729b-4a0e-9c14-b7117d33a817}'
[WinUSB:Info]	Cannot create device for '\\?\usb#vid_256c&pid_006e&mi_00#7&4caab7b&0&0000#{62f12d4c-3431-4efd-8dd7-8e9aab18d30c}'
[RootHub:Info]	Debounced WinUSBRootHub's DevicesChanged event
[RootHub:Info]	Debounced WinUSBRootHub's DevicesChanged event
[RootHub:Info]	Debounced HidSharpDeviceRootHub's DevicesChanged event
[Detect:Info]	Searching for tablets...
[Detect:Info]	No tablets were detected.
